src/seq/index.rs

```diff
@@ -380,7 +380,7 @@ where
 
     #[cfg(not(feature = "nightly"))]
     {
-        use std::collections::BinaryHeap;
+        use alloc::collections::BinaryHeap;
```

Comment:
Binary Heap is implemented in `alloc` so we can downgrade the requirement.

Reply:
Actually we cannot fully downgrade the requirement because `f64::powf` is not stabilized on `core` yet.